pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["pdm-backend"]
+build-backend = "pdm.backend"
+
+[project]
+name = "qnngds"
+version = "0.0.6"
+authors = [
+  { name="A. Jacquillat", email="audrey01@mid.edu" },
+  { name="Z. Jacquillat", email="audrey.jacquillat@gmail.com" }
+]
+maintainers = [
+  {name="A. Jacquillat", email="audrey01@mit.edu" }
+]
+description = "The QNN library for creating gds files"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {file = "LICENSE.txt"}
+keywords = ["phidl", "nanowire_electronics", "nanofabrication", "gds"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering"
+]
+
+dependencies = [
+  "peppercorn",
```

Comment:
what is `peppercorn` needed for? If it's just part of building the docs, and not used for the rest of the project, is there an easy way to specify a different dependency set? If not no big deal, but it'd be nice to not include packages if they aren't needed for most users (not that it really matters cause the package is probably only a megabyte at most lol)